searx/engines/base.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python
+
+"""
+ BASE (Scholar publications)
+
+ @website     https://base-search.net
+ @provide-api yes with authorization (https://api.base-search.net/)
+
+ @using-api   yes
+ @results     XML
+ @stable      ?
+ @parse       url, title, publishedDate, content
+ More info on api: http://base-search.net/about/download/base_interface.pdf
+"""
+
+from lxml import etree
+from urllib import urlencode
+from searx.utils import searx_useragent
+from cgi import escape
+from datetime import datetime
+import re
+
+categories = ['science']
+
+base_url = 'https://api.base-search.net/cgi-bin/BaseHttpSearchInterface.fcgi'\
+           + '?func=PerformSearch&{query}&boost=oa&hits={hits}&offset={offset}'
+
+# engine dependent config
+paging = True
+number_of_results = 10
+
+
+def request(query, params):
+    # shortcuts for advanced search
+    shorcut_dict = {
```

Comment:
@jibe-b what do you mean by this going further than you proposed to add to searx? I think it is enough to change the keys in the dict to `tit:`, `aut:` and so on like described in the Limit search area section of BASE help. Extra API params could be parameterized using keywords you/we come up with. Or maybe we do not need all of the params provided by the BASE API.

What do you think @asciimoo?